### DISCUSSION TITLE
Idea: Allow adding of additional search providers

### DISCUSSION THREAD

**Discussion by collse** [linuxserver/Heimdall#1075] [Ideas]
first of all, thank you for excellent solution but I wish it had the option to manually add an/multiple additional search providers via the UI.

I run Heimdall on docker so I can go into the container and add an additional search provider via  ~/config/www/searchproviders.yaml but having it available from within the UI would make this much easier I would think.

**Comment by Neclony** [comment]
A custom search provider option would clear the problem of manually adding every single one.
Just like web bowsers did:
![image](https://github.com/linuxserver/Heimdall/assets/22119945/0222efb9-a000-4333-8424-c41d05dd461b)
